static/app/components/workflowEngine/layout/detail.tsx

```diff
@@ -20,12 +24,21 @@ export interface WorkflowEngineDetailLayoutProps {
  */
 function DetailLayout({children}: WorkflowEngineDetailLayoutProps) {
   const title = useDocumentTitle();
+  const organization = useOrganization({allowNull: false});
+  const project = useProjectFromSlug({organization, projectSlug: 'javascript'});
   return (
     <StyledPage>
       <Layout.Header>
         <Layout.HeaderContent>
           <BreadcrumbsFromContext />
           <Layout.Title>{title}</Layout.Title>
+          <ProjectContainer>
+            {!project ? (
+              <LoadingIndicator />
+            ) : (
+              <ProjectBadge project={project} disableLink avatarSize={16} />
```

Comment:
just curious - why not have the `ProjectBadge` use the `useProjectFromSlug` hook?

Reply:
Just using the component we have already, but yeah I should make a quick wrapper

Reply:
Actually @ameliahsu [pointed out](https://github.com/getsentry/sentry/pull/85699?email_source&#discussion_r1966288948) that we don't really need the extra network request! Removed in [eae2ccf](https://github.com/getsentry/sentry/pull/85699/commits/eae2ccf1b1a2d352fd0f79b4614422396f6af093) ✨